examples/7SegmentLED_ChangeOrderDigits/7SegmentLED_ChangeOrderDigits.ino

```diff
@@ -33,6 +33,13 @@
 auto display = TM1637_7SegmentLEDController<6 /* The number of digits of 7 segment display */>(
   PIN_DIO,
   PIN_CLK,
+  //By detault you do not need the array specified below. This array allows you to rearrange digits on the 7-segment display. 
+  //Rearranging digits may be needed as some manufatures of the display do not wire them up the same as the library is expecting.  
+  //For example a company that produces a 6 digit 7-segment display when you perform a display.displayNumericalString("123456"); 
+  //the screen will actually display 321654. There are 2 ways to fix this, 1 rewire the 7 segment display if you're using a breadboard
```

Comment:
Thank you! Very helpful😆

I think this sentence...

> For example a company that produces **a 6 digit 7-segment display** when you perform a display.displayNumericalString("123456"); the screen **will** actually display 321654.

...should be better like below

> For example a company that produces **a display as described above** when you perform a display.displayNumericalString("123456"); the screen **may** actually display 321654.

because this is not limited to 6-digit display, and may be the other company's one shows different, like '654321' and so on.
(but I'm not sure if this is correct expression...🤔)

Reply:
all good with latest